### ISSUE TITLE
Make compression style configurable to ensure compatibilty with Webkit

### ISSUE THREAD

**Issue by colszowka** [rails/sass-rails#44]
I just migrated http://beta.ruby-toolbox.com to Rails 3.1 and was already using compass and sass before. I already bumped into this issue before, but it has reappeared now and I can't give you exact reasons for this right now, but the issue is this: 

If CSS is compressed into 1 single line, Webkit browsers (I at least can confirm this on Chrome and Safari) will eventually mess up and only display part of the defined styles. As I said, I can't currently tell you why this happens and exactly when, but it is the case for example when Rails 3.1 and sass-rails precompiled this css into a single line (this is the fixed version of it): http://assets3.ruby-toolbox.com/assets/preview-67460b20d3e23954001364afc2b08da1.css

I also had this issue before Rails 3.1 with plain Compass and had to adjust the compression settings back then as well.

**TL;DR**
- The default of `:style => :compressed` in https://github.com/rails/sass-rails/blob/master/lib/sass/rails/compressor.rb can lead to big and hard to diagnose trouble
- The default is hard-coded, even though there is `config.sass.style` in the Railtie (but maybe that's for development mode? Also, since an instance of CssCompressor is created in the Railtie, config options set _after_ sass-rails being loaded won't be respected, right?)
- Using `:style => :nested` solved the problem for me as the rules have newlines again (see the linked css file). To accomplish this, I had to re-open the CssCompressor class and overwrite the whole thing to change the style.

I'm also not sure if :compressed is so much better than :nested as a default when I think of the potential headaches people could get from this. On the linked (gzipped) CSS file, the difference between those was +0.3 KB on a 6 KB file, so I think it is negligible. But I don't want this to result in a big discussion about CSS compression. It's just that in the current state my CSS breaks on Rails 3.1 and I have to hack around the problem, which is not ideal :)

**Comment by ekaitzht** [comment]
@colszowka could you fix this problem? It's happening to me the same.

**Comment by colszowka** [comment]
@ekaitzht Sorry, not really. I think I moved over to doing sass `include` statements instead of requires, that usually is much better to work with than using sprockets anyway (it also works with plain css files, for example when you import them from bower components), so this went away. Originally I had changed the sass compression method to not compress though, as described in the initial post.

**Comment by ekaitzht** [comment]
@colszowka ok thanks
